horace_core/sqw/PixelData/@PixelData/PixelData.m

```diff
@@ -518,14 +524,16 @@ function delete(obj)
             if size(pixel_data, 1) ~= obj.PIXEL_BLOCK_COLS_
                 msg = ['Cannot set pixel data, invalid dimensions. Axis 1 must '...
                     'have length %i, found ''%i''.'];
-                error('PIXELDATA:data', msg, obj.PIXEL_BLOCK_COLS_, ...
+                error('HORACE:PixelData:invalid_argument', msg, obj.PIXEL_BLOCK_COLS_, ...
                     size(pixel_data, 1));
             elseif ~isnumeric(pixel_data)
                 msg = ['Cannot set pixel data, invalid type. Data must have a '...
                     'numeric type, found ''%s''.'];
-                error('PIXELDATA:data', msg, class(pixel_data));
+                error('HORACE:PixelData:invalid_argument', msg, class(pixel_data));
             end
             obj.raw_data_ = pixel_data;
+            %obj.num_pixels_ = size(pixel_data,2); % breaks filebased
+            %PixelData
```

Comment:
do you want to keep this commented out code?

Reply:
yes. I wanted to implement this operation, but apparently it does not work. To remember that it should not be tried again or more attention paid to implementation, the comment remains.